src/reporter.cc

```diff
@@ -91,6 +91,11 @@ std::string BenchmarkReporter::Run::benchmark_name() const {
 }
 
 double BenchmarkReporter::Run::GetAdjustedRealTime() const {
+  if (use_manual_time && manual_time->customized_units) {
+    double new_time = real_accumulated_time;
```

Comment:
That doesn't look right, it will print `132414 cycles` instead of `132 kcycles` e.g?

Reply:
This is mostly because I'm not sure of if the nano/micro/milli prefixes will apply to whatever the type is, so it's passed through directly (and the printed time_unit value has no prefix). I don't know of a way to signal "and nano / micro / milli / centi / etc. apply here".

Reply:
I made some changes to allow for custom units to be applied to be applied here without checking for customized units or manual time.